bin/install.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+appendIfNotExist() {
+  if ! grep -Fxq "$1" $HOME/.zshrc; then
+    if $2; then
+      echo "Appending $1 to .zshrc"
+    fi
+    echo $1 >> $HOME/.zshrc
+  fi
+}
+
+kitDir=$HOME/.fantastic-kit
+
+# in case user has already installed fk, nuke everything and reinstall
+if [ -d $kitDir ]; then
+  echo "found existing fantastic-kit installation. Removing old files..."
+  rm -rf $kitDir
+fi
+
+
+echo "Downloading fantastic-kit source to $kitDir"
+mkdir -p "$kitDir"
+cd $kitDir
+git clone --depth=1 https://github.com/fantastic-kit/fantastic-kit.git .
+
+appendIfNotExist "######## Setup script setup by fantastic-kit ########" false
+
+# sourcing fk.sh if it is not already added to the .zshrc file
+appendIfNotExist "source \$HOME/.fantastic-kit/bin/fk.sh" true
```

Comment:
![image](https://user-images.githubusercontent.com/4193035/51558115-ab316680-1e4c-11e9-83c2-823a5a56d277.png)

Also you don't need to explicitly say true

Reply:
Updated the function so it no longer requires boolean